### ISSUE TITLE
Add info to comments as well?

### ISSUE THREAD

**Issue by gr2m** [hzoo/contributors-on-github#21]
we create a lot of issues that are reserved for first-time contributors, like this one: https://github.com/hoodiehq/camp/issues/37

It would be great to see on first sight if they sent PRs in the past. But so far the badge gets only added to comment of the author of the issue. Could we change that?

**Comment by hzoo** [comment]
Yeah we could totally do that - I didn't do it initially since we didn't have to + more requests

**Comment by gr2m** [comment]
I wonder if we could add a passive state which would just be a button saying "load contribution stats" or something similar. We could add that to all comments because no requests are sent initially at all, only when clicking that button. We could still directly load the stats for the first comment but I’d actually prefer to have the button and only load stats if I need them. What do you think?

**Comment by koppor** [comment]
I would like to ask for possibilities here - we host a world-wide education project and would like to have an indication whether the contributor is here for GSoC or from a university or for other reasons. The # of PRs is a good indication.

Currently, there is nothing shown on a comment:

<img width="946" height="227" alt="Image" src="https://github.com/user-attachments/assets/aea23fea-c006-4c64-b818-a9b57c8d376b" />

Since the issue is a good **second** issue, I would like to check if the contributor worked on a good **first** issue before.

**Comment by koppor** [comment]
Workaround could be to include the statistics in a welcoming message for new contributors: https://github.com/takanome-dev/assign-issue-action/issues/406
